prisma/seed.js

```diff
@@ -0,0 +1,15 @@
+const { PrismaClient } = require('@prisma/client');
```

Comment:
@nathanSamosa what is the exact purpose of the seed.js file?

Reply:
It is to populate the database whenever the schema is updated or the data is reset. It is currently empty in our project since we are populating the pokemon in the controller, but if we want some default users, like a demo account or admin, we can do that inside the seed.js so they always exist after the database is reset.